.freeCodeCamp/tooling/validate.js

```diff
@@ -112,5 +113,62 @@ export async function validateCurriculum() {
       );
     }
   }
+
+  for (const seed of seeds) {
+    const project = seed.replace('-seed.md', '.md');
+    const projectDirPath = project.replace('.md', '');
+    try {
+      await access(projectDirPath, constants.F_OK);
+    } catch (e) {
+      throw new Error(
+        `Seed "${seed}" does not have an associated root directory`
+      );
+    }
+
+    const mdPath = `${CURRICULUM_PATH}/${project}`;
```

Comment:
Please use `join` here. I get scared of errors popping up when one of these paths randomly begins/ends with a `/`